Xamarin.Forms.Platform.UAP/Shell/ShellRenderer.cs

```diff
@@ -42,6 +42,10 @@ public ShellRenderer()
 				PaneClosing += (s, e) => OnPaneClosed();
 			if (ApiInformation.IsEventPresent("Windows.UI.Xaml.Controls.NavigationView", "PaneOpening"))
 				PaneOpening += (s, e) => OnPaneOpening();
+			if (ApiInformation.IsEventPresent("Windows.UI.Xaml.Controls.NavigationView", "PaneOpened"))
+				PaneOpened += (s, e) => OnPaneOpened();
```

Comment:
The changes looks nice but, could we add the unsubscription of these events? Maybe using `OnElementSet`, if `_shell` exists?

Reply:
👍